cmd/collectors/rest/plugins/sensor/sensor.go

```diff
@@ -47,7 +47,7 @@ var ambientRegex = regexp.MustCompile(`^(Ambient Temp|Ambient Temp \d|PSU\d AmbT
 var powerInRegex = regexp.MustCompile(`^PSU\d (InPwr Monitor|InPower|PIN|Power In)$`)
 var voltageRegex = regexp.MustCompile(`^PSU\d (\d+V|InVoltage|VIN|AC In Volt)$`)
 var currentRegex = regexp.MustCompile(`^PSU\d (\d+V Curr|Curr|InCurrent|Curr IIN|AC In Curr)$`)
-var eMetrics = []string{"power", "ambient_temperature", "max_temperature", "average_temperature", "average_fan_speed", "max_fan_speed", "min_fan_speed"}
+var eMetrics = []string{"power", "average_ambient_temperature", "min_ambient_temperature", "max_temperature", "average_temperature", "min_temperature", "average_fan_speed", "max_fan_speed", "min_fan_speed"}
```

Comment:
minor:we have enough of these that it probably makes sense to put each on a separate line and sort